array-filter/main.js

```diff
@@ -0,0 +1,48 @@
+const numbers = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const names = [
+  'Ada',
+  'Hedy',
+  'Jean',
+  'Grace',
+  'Evelyn',
+  'Joan',
+  'Elizabeth',
+  'Janese',
+  'Donna'
+];
+const evenNumbers = [];
+numbers.filter(even => {
+  if (even % 2 === 0) {
+    evenNumbers.push(even);
+  }
+  return evenNumbers;
+});
+console.log(evenNumbers);
+
+const overFive = [];
+numbers.filter(over => {
+  if (over > 5) {
+    overFive.push(over);
+  }
+  return overFive;
+});
```

Comment:
Please review the documentation again. `Array.prototype.filter` creates a new array depending on whether or not your callback returns truthy or falsy. Don't push to a different array.